smarts/core/smarts.py

```diff
@@ -226,6 +226,9 @@ def _step(self, agent_actions):
         # Agents
         self._agent_manager.step_agent_sensors(self)
 
+        # Panda3D
+        self.taskMgr.mgr.poll()  # update camera position in the render pipeline
+
```

Comment:
This is the change we brainstormed as a team, I thought we agreed that we had to use task manager to be able to synch update and render threads per frame?
If we do not use task manager to synch per frame, there can be mismatch across multiple steps.
@JVillella